google-ads/pom.xml

```diff
@@ -16,7 +16,9 @@
  * specific language governing permissions and limitations
```

Comment:
While you're in here, could you update this copyright as well?

https://github.com/google/new-project#source-code-headers